docs/usage/self-hosted-configuration.md

```diff
@@ -11,6 +11,10 @@ Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md
 
 ## allowCustomCrateRegistries
 
+## allowGitLfs
+
+**Note** You need to also configure `gitNoVerify` to allow commit and push git lfs objects.
```

Comment:
I feel like this text is missing something?

How about:

```suggestion
**Note** You need to also configure `gitNoVerify` to `true` to allow Renovate to commit and push Git LFS objects.
```

I think the correct way to capitalize is `Git LFS` by the way. 😉

Reply:
Sure, but `gitNoVerify` is a string array option, so no `true` here

Reply:
Updated my suggestion. 😄